src/metric/populate_test.go

```diff
@@ -144,24 +143,8 @@ func TestPopulateK8s(t *testing.T) {
 
 	k8sVersion := &version.Info{GitVersion: "v1.15.42"}
 	err = p.Populate(foo, kubeletSpecs, i, "test-cluster", k8sVersion)
-	assert.Error(t, err)
+	assert.Empty(t, err.(data.PopulateResult).Errors)
```

Comment:
Wouldn't this assert pass if the type assertion fails?

Reply:
It will panic if type assertion fails. Perhaps we should assert the type explicitly above, so the test suite does not panic.

Reply:
I added `require.IsType` before we assert, so we don't panic.